ddtrace/_trace/span.py

```diff
@@ -670,7 +671,7 @@ def _validate_attribute(self, key: str, value: object) -> bool:
         first_type = type(value[0])
         for val in value:
             if not isinstance(val, first_type) or not self._validate_scalar(key, val):
-                log.warning("record_exception: Attribute %s array must be homogenous: %s.", key, value)
+                log.warning("record_exception: Attribute %s array must be homogeneous: %s.", key, value)
```

Comment:
are changes to this file needed at all?

this is a good spelling fix, but seems like you don't need to update this file at all?